koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -376,6 +378,13 @@ def _incremental_upload_logs(self, child_pid=None):
         finally:
             watcher.clean()
 
+    def _upload_logs_once(self):
+        """Upload log updates without waiting for anything"""
+        try:
+            self._incremental_upload_logs()
+        except koji.ActionNotAllowed:
```

Comment:
Maybe I need explanation why. (Shall we log it when this happens, at leas for debugging on koji builder side?)

Reply:
reopen_file raises that https://github.com/containerbuildsystem/koji-containerbuild/blob/master/koji_containerbuild/plugins/builder_containerbuild.py#L186

when it gets OSError, and there isn't anymore info anyway

I am basically doing the same like https://github.com/containerbuildsystem/koji-containerbuild/blob/master/koji_containerbuild/plugins/builder_containerbuild.py#L505

Reply:
ok